Composer/packages/client/src/components/Notifications/useSurveyNotification.ts

```diff
@@ -55,6 +57,7 @@ export const useSurveyNotification = () => {
               // This is safe; we control the URL that gets built
               // eslint-disable-next-line security/detect-non-literal-fs-filename
               window.open(url, '_blank');
+              surveyStorage.set('epochLastTaken', Date.now());
```

Comment:
What is `epochLastTaken`?

Reply:
You should also put this string in a variable in the surveyStorage file and import the variable to read/write to localstorage

Reply:
It's the timestamp for when the survey window was last opened (which is when we presume the user took the survey). It gets compared to Date.now() inside the hook so we can tell if it's been long enough to ask the user to take a survey again. Since this wasn't getting set, though, that logic assumed the user just never took the survey at all and so that check always passed (as if it were the first time the survey showed up).

Reply:
Is there a more disruptive key we could use? lastSurveyTakenOnTimestamp